server/src/core/auth/services/token.service.ts

```diff
@@ -86,6 +86,36 @@ export class TokenService {
     };
   }
 
+  async generateApiKeyToken(
+    workspaceId: string,
+    apiKeyId: string,
+    expiresAt?: Date | string,
+  ): Promise<AuthToken> {
+    const secret = this.environmentService.getLoginTokenSecret();
+    let expiresIn: string | number;
+    let tokenExpiryDate: Date;
```

Comment:
@martmull Instead of `tokenExpiryDate` maybe we can name it `expirationDate`

Reply:
ok